charts/influxdb-enterprise/README.md

```diff
@@ -0,0 +1,128 @@
+# InfluxDB Enterprise
+
+## QuickStart
+
+```bash
+helm repo add influxdata https://helm.influxdata.com/
+helm upgrade --install influxdb influxdata/influxdb-enterprise --namespace monitoring
+```
+
+> **Tip**: `helm upgrade --install [RELEASE] [CHART] [FLAGS]` can be shortened : `helm upgrade -i [RELEASE] [CHART] [FLAGS]`
+
+## Introduction
+
+This chart bootstraps an InfluxDB Enterprise cluster, with a StatefulSet for both the Meta and Data nodes.
+
+## Prerequisites
+
+- Kubernetes 1.4+
+- PV provisioner support in the underlying infrastructure (optional)
+
+### Secrets
+
+This chart REQUIRES some mandatory secrets in-order to function.
```

Comment:
Maybe:

"This chart requires the following secrets in order to function:

[list or description of secrets]"